fbw-a32nx/src/systems/instruments/src/EWD/PseudoFWC.ts

```diff
@@ -981,6 +988,18 @@ export class PseudoFWC {
       () => (this.auralSingleChimePending = false),
       PseudoFWC.AURAL_SC_INHIBIT_TIME,
     );
+
+    this.acESSBusPowered.sub((v) => {
+      if (v) {
+        this.startupTimer.schedule(() => {
+          this.startupCompleted.set(true);
+          console.log('PseudoFWC startup completed.');
+        }, PseudoFWC.FWC_STARTUP_TIME);
+      } else {
```

Comment:
Nitpick: please clear the timer in case someone decides to toggle the power multiple times ;)

Reply:
I don’t think I have to clear the timer, I can just schedule it again. In case AC ESS is disconnected, I’m already resetting the startup variable to false. I tried it, re-cycling works.

Reply:
Wouldnt in a situation where you power on the FWC and then quickly power it off again, the startupComplete var be first set to false because of line 999 but then after the timer has elapsed be turned to true ?

Reply:
We discussed it briefly on discord and agreed on that as well. I think floridude wants to change it 👍

Reply:
thanks for spotting that, just pushed a fix :)